internal/config/legacy.go

```diff
@@ -1,329 +1,44 @@
-	Version     string            `yaml:"version"`
-
-	// Catches all undefined files and must be empty after parsing.
-	XXX map[string]any `yaml:",inline"`
-}
-
-// CheckOverflow implements configer.
-func (c *Pre140) CheckOverflow() error {
-	return checkOverflow(c.XXX)
-}
-
-// Config converts the Pre140 config to the current config struct.
-func (c *Pre140) Config() *Config {
-	cfg := &Config{
-		AutoImport:  c.AutoImport,
-		ClipTimeout: c.ClipTimeout,
-		ExportKeys:  true,
-		Parsing:     true,
-		Path:        c.Path,
-		SafeContent: c.SafeContent,
-		Mounts:      make(map[string]string, len(c.Mounts)),
 	}
-
-	for k, v := range c.Mounts {
-		cfg.Mounts[k] = v
-	}
-
-	return cfg
-}
-
-// Pre130 is the gopass config structure before version 1.3.0. Not all fields were.
-// available between 1.0.0 and 1.3.0, but this struct should cover all of them.
-type Pre130 struct {
-	AlwaysTrust bool              `yaml:"alwaystrust"` // always trust public keys when encrypting.
-	AskForMore  bool              `yaml:"askformore"`  // ask for more data on generate.
-	AutoImport  bool              `yaml:"autoimport"`  // import missing public keys w/o asking.
-	AutoPull    bool              `yaml:"autopull"`    // pull from git before push.
-	AutoPush    bool              `yaml:"autopush"`    // push to git remote after commit.
-	ClipTimeout int               `yaml:"cliptimeout"` // clear clipboard after seconds.
-	Debug       bool              `yaml:"debug"`       // enable debug output.
-	LoadKeys    bool              `yaml:"loadkeys"`    // load missing keys from store.
-	Mounts      map[string]string `yaml:"mounts,omitempty"`
-	NoColor     bool              `yaml:"nocolor"`     // disable colors in output.
-	Confirm     bool              `yaml:"noconfirm"`   // do not confirm recipients when encrypting.
-	Path        string            `yaml:"path"`        // path to the root store.
-	PersistKeys bool              `yaml:"persistkeys"` // store recipient keys in store.
-	SafeContent bool              `yaml:"safecontent"` // avoid showing passwords in terminal.
-	Version     string            `yaml:"version"`
-
-	// Catches all undefined files and must be empty after parsing.
-	XXX map[string]any `yaml:",inline"`
-}
-
-// CheckOverflow implements configer.
-func (c *Pre130) CheckOverflow() error {
-	return checkOverflow(c.XXX)
-}
-
-// Config converts the Pre130 config to the current config struct.
-func (c *Pre130) Config() *Config {
-	cfg := &Config{
-		AutoImport:  c.AutoImport,
-		ClipTimeout: c.ClipTimeout,
-		ExportKeys:  true,
-		Parsing:     true,
-		Path:        c.Path,
-		SafeContent: c.SafeContent,
-		Mounts:      make(map[string]string, len(c.Mounts)),
-	}
-
-	for k, v := range c.Mounts {
-		cfg.Mounts[k] = v
-	}
-
-	return cfg
-}
-
-func pathFromURL(u string) (string, error) {
-	if !strings.Contains(u, "://") {
-		return u, nil
+	for alias, path := range cfg.Mounts {
+		if err := c.SetGlobal(mpk(alias), path); err != nil {
```

Comment:
What if different mounts have the same alias?

Reply:
That's an invalid setup. I mean ... the mounts map is keyed by this alias. So I'd say that can't happen.